bundles/statistics/statsgrid2016/resources/scss/style.scss

```diff
@@ -959,9 +959,12 @@ background-color: #888888;
 /* ********************************************
  * Indicator list
  * ********************************************/
- ol.statsgrid-indicator-list li:nth-child(2n) {
-    background-color: #f3f3f3;
-}
+ ol.statsgrid-indicator-list {
+    min-width: 430px !important;
```

Comment:
Check if !important is really required. Otherwise looks good for merging.

Reply:
Removed !important since it is not required.